src/xlsx/extraction/chart_extractor.ts

```diff
@@ -29,7 +29,8 @@ export class XlsxChartExtractor extends XlsxBaseExtractor {
           title: chartTitle,
           type: CHART_TYPE_CONVERSION_MAP[chartType]!,
           dataSets: this.extractChartDatasets(
-            this.querySelector(rootChartElement, `c:${chartType}`)!
+            this.querySelector(rootChartElement, `c:${chartType}`)!,
+            chartType
           ),
           labelRange: this.extractChildTextContent(rootChartElement, "c:ser c:cat c:f"),
```

Comment:
this line also needs to be adapted. With the scatter chart, the label range is set as a `numRef` and not `cat` so the selector should be `c:ser c:numRef c:f`